http-lambda-invoker.go

```diff
@@ -59,6 +62,17 @@ func getConfig(key string) string {
 	}
 }
 
+func makeProxyHeaders(original_headers map[string][]string) proxyHeader {
+	var new_headers proxyHeader
+	new_headers = make(proxyHeader)
+
+	for header := range original_headers {
+		new_headers[header] = strings.Join(original_headers[header], "")
+	}
+
+	return new_headers
+}
```

Comment:
Logically looks correct, but go-lint says `don't use underscores in Go names; func parameter original_headers should be originalHeaders`.

Also I think we could go with a shorter declaration:

`var newHeaders = make(proxyHeader)`

Reply:
Duly noted and updated.